src/writer.ts

```diff
@@ -354,6 +356,26 @@ export const handlePropose: CheckpointWriter = async ({ block, tx, rawEvent, eve
     await user.save();
   }
 
+  const evmSlotValueStrategyIndex = space.strategies.findIndex(
+    strategy =>
+      validateAndParseAddress(strategy) ===
+      validateAndParseAddress(networkProperties.evmSlotValueStrategyAddress)
+  );
+
+  if (evmSlotValueStrategyIndex !== -1) {
+    const [l1TokenAddress] = space.strategies_params[evmSlotValueStrategyIndex].split(',');
+
+    try {
+      await registerProposal({
```

Comment:
Wouldn't this run everytime we sync the API? https://github.com/snapshot-labs/mana/pull/125/files#diff-e81a71d8f28ec2bd9560d90d23bbda701b5c6dadad01b5f0ad44984d5d92f989R59

Reply:
It will, I think we should prevent handling those twice at Mana level by storing those in DB as we can't really control it on sx-api.

Reply:
Another option would be to keep track which block was indexed already once on SX-API with some value that not get deleted when we reset SX-API. This system could be used later to have a general purpose webhook system to send notification for new proposal or vote in future (notifications is a new feature request from Starknet).

Reply:
But even with this I imagine we still need to prevent Mana to send the request / tx twice.

Reply:
For now I added just guard on Mana. This feature is already pretty big with multiple connected PRs and adding webhook system to Checkpoint would push it a bit too far in my opinion.
We can work on webhooks support for checkpoint separately with proper approach from the start.

Reply:
I dont think the webhook system should be part of Checkpoint or SX-API, SX-API would just trigger info on an external API. The external API can be a webhook provider like webhook.site and will be responsible to keep list of suscribers and sending webhook notification at the right time and with retries.